vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
**praise**: Including a link to the Vite configuration documentation is a great practice for helping new developers understand the setup.